misc/kubernetes-examples/blog1/mysql-deployment-vsphere.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
```

Comment:
@shuklanirdesh82 Thanks for mentioning about this. I have created branch on vmware/kubernetes and protected the branch.